vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -416,6 +420,36 @@ private void unstashWorkspace(config, prefix) {
     }
 }
 
+private List getInitContainerList(config){
+    def initContainerSpecList = []
+    if (config.initContainerImage && config.containerMountPath) {
+        def initContainerName = config.initContainerImage.toLowerCase()
+        def initContainerSpec = [
+            name           : config.initContainerName,
+            image          : config.initContainerImage,
+            volumeMounts   : [[name: "volume", mountPath: config.containerMountPath]],
+        ]
+        if (config.initContainerCommand == null) {
+            echo "Please provide initContainerCommand for ${config.initContainerImage}"
```

Comment:
If this is not an error case then why does one **have** to provide an `initContainerCommand`? Maybe it would be better to write here that no `initContainerCommand` was provided, so the default will be used.